README.md

```diff
@@ -14,27 +14,207 @@ Choosing between the two formats is up to you and your preferences.
 
 If you already have some experience in React Native, Flutter, or Swift you may find some similarities. MauiReactor borrows some implementation designs from them while maintaining the normal MAUI application development you're used to.
 
-NO XAML needed, just using C# you can write fluent declarative UI with MauiReactor components:
-
-```csharp
-class MainPage : Component
-{
-    public override VisualNode Render()
-        => ContentPage("Login",
-            VStack( // Vertical Stack
-                Label("User:"),
-                Entry(),
-                Label("Password:"),
-                Entry(),
-                HStack( // Horizontal Stack
-                    Button("Login"),
-                    Button("Register")
-                )
-            )
-            .Center()
-        );
-}
-```
+NO XAML needed, just using C# you can write fluent declarative UI with MauiReactor components.
+
+Let's build our first Component based app from scratch, so you get the flavor of working with MauiReactor.  
+This app will be focused on teaching, so is quite incomplete.  
+For example, no exception handling, no reading of the user name or password entered.  Several useless buttons.
+
+1. Install MauiReactor templates to Visual Studio 2022 *(if not already installed)*
```

Comment:
We should try not to focus only on Visual Studio 2022 and Windows, the same template can be adopted on other IDEs and platforms